docs/rules/arrow-body-style.md

```diff
@@ -20,15 +20,17 @@ The second one is an object for more fine-grained configuration when the first o
 "arrow-body-style": ["error", "always"]
 ```
 
-When the rule is set to `"always"` the following patterns are considered problems:
+### always
+
+Examples of **incorrect** code for this rule with the `"always"` option:
 
 ```js
 /*eslint arrow-body-style: ["error", "always"]*/
 /*eslint-env es6*/
 let foo = () => 0;
```

Comment:
I'd really prefer to leave the empty line stuff to a subsequent sweep of the files. The full focus of this sweep was to get the example headings/captions much more consistent. What do you think?

BTW, I'm also struggling to work out how to do separate PRs for separate branches - it seems to always want to merge them. As such I've got another 3 or 4 batches of files sat waiting on this PR going through (the entirety of the es6 rules has now had initial heading/caption sweep).